packages/compiler/test/expression_parser/parser_spec.ts

```diff
@@ -935,8 +935,7 @@ function expectBindingError(text: string, message: string) {
 }
 
 /**
- * Check that an malformed action parses to a recovered AST while emitting an
- * error.
+ * Check that an malformed action parses to a recovered AST while emitting an error.
```

Comment:
Haha, I meant to fix the type 😅 
```suggestion
 * Check that a malformed action parses to a recovered AST while emitting an error.
```

Reply:
:(: done